migrations/versions/e73480955f62_.py

```diff
@@ -0,0 +1,126 @@
+"""Schedule event to refresh monthly leaderboard
+
+Revision ID: e73480955f62
+Revises: 8b81804f8861
+Create Date: 2019-08-07 21:54:52.759443
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e73480955f62'
+down_revision = '8b81804f8861'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute("""
+
+    CREATE EVENT IF NOT EXISTS evt_refresh_dwh_leaderboard
+    ON SCHEDULE
+        EVERY 3 MINUTE
+    COMMENT 'Refresh monthly leaderboard'
+    DO
+    BEGIN
+        SET @service_type = 'openvpn',
```

Comment:
Why we account only openvpn service? Wireguard can run behind residential IP too.

Reply:
no idea, keeping existing functionality